src/js/view/preview.js

```diff
@@ -30,6 +35,25 @@ Object.assign(Preview.prototype, {
             backgroundImage: backgroundImage
         });
     },
+    enableZoom: function() {
+        if (this.model.get('state' !== 'idle')) {
+            return;
+        }
+        
+        setTimeout(() => {
+            const originX = Math.ceil(Math.random() * 100) + '%';
+            const originY = Math.ceil(Math.random() * 100) + '%';
+            
+            this.el.classList.add('jw-ab-zoom-thumbnail');
+            this.el.style.transformOrigin = originX + ' ' + originY;
```

Comment:
So we’re changing origin every 2 s and zooming over 10, just want to be sure that’s expected (for the origin to change mid animation)

Reply:
Hmmm. That shouldn't happen. The origin should only be set once, but there is a 2s delay before the animation begins.

Reply:
Ah! It's a timeout, not an interval - my bad.

Reply:
we should set the result of this timeout to a variable and `clearTimeout` in the `destroy` function to 4ensure this doesn't get run after destroy